cli/bin/run.js

```diff
@@ -1,13 +1,16 @@
+const path = require('path')
 const { execSync } = require('child_process');
 
 let runtime = require('../../runtime/dist/grain-runtime.js');
 let grainRoot = execSync('grain-root').toString('utf8').trim();
 
-module.exports = async function run(path, options) {
+module.exports = async function run(filename, options) {
   try {
-    let locator = runtime.defaultFileLocator(options.includeDirs.concat(`${grainRoot}/lib/grain/stdlib`));
+    let base_path = path.dirname(filename)
+    let includeDirs = [base_path].concat(options.includeDirs, `${grainRoot}/lib/grain/stdlib`)
```

Comment:
Is this functionally the same as the original line? Methinks not, unless either `concat` is smarter than I think or you changed the type of `includeDirs`

Reply:
It's the same. `concat` takes any number of arrays or values.

Would it read easier if it were
```js
let includeDirs = [base_path, ...options.includeDirs, `${grainRoot}/lib/grain/stdlib`]
```
?

Reply:
@ospencer For me, I think that would be more readable, yeah.